vocabularies/Common.xml

```diff
@@ -1454,6 +1454,105 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <!--
+      <Term Name="RecommendationsFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#GetRecommendations))" />
+      </Term>
+      -->
+      <Function Name="GetRecommendations" IsBound="true">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.IsSignature" />
+        <Annotation Term="Core.Description" String="Computes AI-based recommendations for the bound entity and its related entities" />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="Computes recommendations for the bound entity and its related entities" />
```
AI-based is cool, but do we want to restrict use of this term to special implementations?

Reply:
Our AI ethics demand that we let people know whether they are talking to AI or not.